src/main/java/org/reaktivity/nukleus/amqp/internal/stream/AmqpServerFactory.java

```diff
@@ -1027,7 +1107,7 @@ private int decodeTransfer(
                 {
                     break decode;
                 }
-                server.decodableBodyBytes -= transfer.sizeof();
+                server.decodableBodyBytes -= transfer.sizeof() + PERFORMATIVE_SIZE;
```

Comment:
Why did this change?

Reply:
Because we assign the value of `decodableBodyBytes` as
`server.decodableBodyBytes = frameSize - frameHeader.doff() * 4;`
and this includes performative (3 bytes), `TRANSFER` list and payload.

And so I wanted to subtract the amount of the performative bytes (3 bytes)